README.md

```diff
@@ -10,6 +10,9 @@ Ambassador Edge Stack [![Build Status][build-status]][build-pages]
 [slack-url]:     https://a8r.io/slack
 [slack-join]:    https://img.shields.io/badge/slack-join-orange.svg
 
+> **Disclaimer**: This repository is in a read only state. We do not monitor issues or pull requests. Questions and issues should either be direct to the [Ambassador Labs community slack](https://a8r.io/slack) or the [Emissary-ingress](https://github.com/emissary-ingress/emissary/) repository. <br/> Information regarding specific edge-stack releases can be found in matching `rel/{release version}` branches. Example Edge-stack v3.2.0 information is at [rel/v3.2.0](https://github.com/datawire/edge-stack/tree/rel/v3.2.0).
```

Comment:
Rather than make this a Quote, do we want to just gut the content below and make this Disclaimer bold and the only focused content?

Reply:
This was my initial thought. We remove everything from the readme and just leave this text. Wasn't sure if that is too invasive.

I can do just that